.github/workflows/release.yaml

```diff
@@ -12,7 +12,7 @@ jobs:
       id-token: write
     uses: ./.github/workflows/reuse.yaml
     with:
-      publish_command: ko-build-initContainer
+      publish_command: ko-publish-initContainer
```

Comment:
Are we ready to test the release process with all the `ko` changes?

Reply:
but `ko-publish-initContainer` looks wrong, should be `ko-publish-kyvernopre`.
let me check

Reply:
fixed